libMagAOX/app/MagAOXApp.hpp

```diff
@@ -1079,7 +1115,7 @@ class MagAOXApp : public application
 
     INDI_SETCALLBACK_DECL( MagAOXApp, m_indiP_powerChannel );
 
-    ///@} Power Management
+    ///@} --Power Management
```

Comment:
?

Reply:
I think the **--** has meaning in whatever documentation tool parses the **@}** but I am okay with removing this fix.

Reply:
Also note that the earlier **@(** is missing in the base file:  do we still want that fixed here?

Reply:
undid both of these comments.